docs/articles/documentation/using-testcafe/programming-interface/TestCafe.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: docs
+title: TestCafe Class
+permalink: /documentation/using-testcafe/programming-interface/TestCafe/
+---
+# TestCafe Class
+
+A TestCafe server instance.
+
+To create it, use the [createTestCafe](/testcafe/documentation/using-testcafe/programming-interface/global/#createTestCafe) global function.
+
+```js
+var createTestCafe = require('testcafe');
+var testCafe = await createTestCafe('localhost', 1337, 1338);
+```
+
+## Methods
+
+### <a class="anchor" name="createBrowserConnection"></a>createBrowserConnection() → [BrowserConnection](/testcafe/documentation/using-testcafe/programming-interface/BrowserConnection/)
+
+Creates a [remote browser connection](/testcafe/documentation/using-testcafe/programming-interface/BrowserConnection/).
+
+#### Example
+
+```js
+var createTestCafe = require('testcafe');
+var testCafe = await createTestCafe('localhost', 1337, 1338);
+
+var remoteConnection = testcafe.createBrowserConnection();
+
+// Outputs remoteConnection.url to visit it from the remote browser.
+console.log(remoteConnection.url);
+remoteConnection.on('ready', () => {
```

Comment:
separate statements with the blank line here. Use `once` for the `ready` event instead of `on`